### ISSUE TITLE
[Bug]: Unable to use priority acces

### ISSUE THREAD

**Issue by jeltehomminga** [wallabyjs/console-ninja#356]
### What happened?

I was expecting to be able to use priority acces as PRO user but instead cannot find a way to make use of this

Since a couple of months(?) console ninja stopped working for NextJs, showing messages like:

```
✘ next.js v14.2.17 is not yet supported in the Community edition of Console Ninja.
We are working hard on it for you https://tinyurl.com/3h9mtwra.

Estimated release dates:
  - Community users: around 5th January, 2025 (subject to team availability)
  - PRO users:       priority access is available now
```

I am a PRO user, but there is no explanation to be found to enable this priority acces. The url goes to a subscription page. I I try to signup again I get the message I am already signed up. Also not clear why it's not enabled by default for pro users

I would expect at least something like a url or explanation how to enable the prio acces

### Version

v1.0.368

### CLI command to start your dev tool

npm run dev

### Steps to reproduce (or sample repo)

npm run dev

### Log output

```shell
✘ next.js v14.2.17 is not yet supported in the Community edition of Console Ninja.
We are working hard on it for you https://tinyurl.com/3h9mtwra.

Estimated release dates:
  - Community users: around 5th January, 2025 (subject to team availability)
  - PRO users:       priority access is available now
```

**Comment by ArtemGovorov** [comment]
The priority access is enabled to all active Pro subscribers, no special action is required. 

Can you please run `Console Ninja: Activate License` command in VS Code, and check the opened view details to make sure your subscription is activated correctly? Once it is activated, please restart your next.js app, and it should work for you.

**Comment by jeltehomminga** [comment]
that says that my subscription is expired and the resume subscription button brings me to a page which doesn't offer an option to resume my subscription for console ninja 

![CleanShot 2024-11-15 at 08 11 58](https://github.com/user-attachments/assets/0f9aa405-7672-4ab6-a7b0-abc52d8fe25e)

If I go to console ninja website and try to purchase POR memberschip again I get an error that the email is already registered

![CleanShot 2024-11-15 at 08 12 45](https://github.com/user-attachments/assets/ca82da57-0461-4452-b69a-fcebcba8d75c)

**Comment by smcenlly** [comment]
@jeltehomminga - the message about your 1 x expired license is for your expired Quokka.js license (I assume you need to scroll down a little to see it):

<img width="988" alt="image" src="https://github.com/user-attachments/assets/bc6ef95a-50d2-42cb-9040-3a8481a83bea">

Your Console Ninja license is valid, and should be working for you.

Could you please close VS Code and then delete your hidden `~/.console-ninja` folder in your home directory. Once you've done this, please try activating again using your email address. If that doesn't work for you, you should be able to use the `Offline License Keys` option in your Wallaby Account and use the license key instead of your email address on the activation screen.

**Comment by smcenlly** [comment]
If you're still having problems after trying those steps, can you please provide us with your `~/.console-ninja/trace.log` file?

**Comment by jeltehomminga** [comment]
Unfortunately that didnt work, also I cannot find an activation screen where I can enter my activation key, do you have a link to that screen?

this is the content of my ~/.console-ninja/trace.log 

```
08:33:25.491 info extension stopping host
08:33:50.927 info extension stopping host
08:33:51.973 info host ------------ starting ------------
08:33:52.096 info host worker Starting Host Worker...
08:33:52.666 info extension host process output: port:[54007]
08:33:52.666 info extension 
08:33:53.659 info host client connected: extension { address: '127.0.0.1', family: 'IPv4', port: 54007 }
08:33:53.738 info host starting host: {"extensionFolder":"/Users/jeltehomminga/.cursor/extensions/wallabyjs.console-ninja-1.0.369","runtimeHookFile":"/Users/jeltehomminga/.cursor/extensions/wallabyjs.console-ninja-1.0.369/out/runtimeHook/tracer.js","globalErrorHandlerFile":"/Users/jeltehomminga/.cursor/extensions/wallabyjs.console-ninja-1.0.369/out/runtimeHook/errorHandler.js","networkLoggingHandlerFile":"/Users/jeltehomminga/.cursor/extensions/wallabyjs.console-ninja-1.0.369/out/runtimeHook/networkLoggingHandler.js","filesToInstrument":[{"path":"/users/jeltehomminga/coding/my-luminus-web/src/components/chart/graph/index.tsx","reason":"search"},{"path":"/users/jeltehomminga/coding/my-luminus-web/src/hooks/usemeterreadingsforean/index.ts","reason":"search"},{"path":"/users/jeltehomminga/coding/my-luminus-web/src/hooks/usemutatemoverequest/usemutatemoverequest.spec.ts","reason":"search"},{"path":"/users/jeltehomminga/coding/my-luminus-web/src/hooks/usecookiereducer/index.tsx","reason":"search"},{"path":"/users/jeltehomminga/coding/my-luminus-web/src/hooks/usemutatedetailsofyourmove/index.tsx","reason":"search"},{"path":"/users/jeltehomminga/coding/my-luminus-web/src/utils/decodejwt.ts","reason":"search"},{"path":"/users/jeltehomminga/coding/my-luminus-web/src/utils/queryclient.ts","reason":"search"},{"path":"/users/jeltehomminga/coding/my-luminus-web/src/utils/getaccountidfromaccesstoken.ts","reason":"search"},{"path":"/users/jeltehomminga/coding/my-luminus-web/src/utils/setupappinsights.ts","reason":"search"},{"path":"/users/jeltehomminga/coding/my-luminus-web/cache-handler.mjs","reason":"search"},{"path":"/users/jeltehomminga/coding/my-luminus-web/public/mockserviceworker.js","reason":"search"},{"path":"/users/jeltehomminga/coding/my-luminus-web/load-appinsights.js","reason":"search"},{"path":"/users/jeltehomminga/coding/my-luminus-web/app/change-email-action.ts","autoLogPoints":[{"range":{"startLine":46,"endLine":46,"startColumn":0,"endColumn":0}}],"reason":"autoLogPoints"},{"path":"/users/jeltehomminga/coding/my-luminus-web/src/screens/my-profile/change-email/change-email.tsx","autoLogPoints":[{"range":{"startLine":75,"endLine":75,"startColumn":0,"endColumn":0}}],"reason":"autoLogPoints"},{"path":"/users/jeltehomminga/coding/my-luminus-web/src/screens/contracts-and-services/contract-overview/contract-details-table/contract-details-table.tsx","autoLogPoints":[{"range":{"startLine":36,"endLine":36,"startColumn":0,"endColumn":0}}],"reason":"autoLogPoints"},{"path":"/users/jeltehomminga/coding/my-luminus-web/mocks/data/current-contract-usps.ts","autoLogPoints":[{"range":{"startLine":74,"endLine":74,"startColumn":0,"endColumn":0}}],"reason":"autoLogPoints"},{"path":"/users/jeltehomminga/coding/my-luminus-web/src/components/chart/graph/helpers.ts","autoLogPoints":[{"range":{"startLine":63,"endLine":63,"startColumn":0,"endColumn":0}}],"reason":"autoLogPoints"},{"path":"/users/jeltehomminga/coding/my-luminus-web/src/hooks/usegetmetersconsumptionsources/index.ts","autoLogPoints":[{"range":{"startLine":79,"endLine":79,"startColumn":0,"endColumn":0}}],"reason":"autoLogPoints"},{"path":"/users/jeltehomminga/coding/my-luminus-web/src/components/chart/period-selector/index.tsx","autoLogPoints":[{"range":{"startLine":39,"endLine":39,"startColumn":0,"endColumn":0}}],"reason":"autoLogPoints"},{"path":"/users/jeltehomminga/coding/my-luminus-web/src/hooks/usemutatephonenumber/index.tsx","autoLogPoints":[{"range":{"startLine":23,"endLine":23,"startColumn":0,"endColumn":0}},{"range":{"startLine":26,"endLine":26,"startColumn":0,"endColumn":0}}],"reason":"autoLogPoints"},{"path":"/users/jeltehomminga/coding/my-luminus-web/app/[lang]/(client)/layout.tsx","autoLogPoints":[{"range":{"startLine":174,"endLine":174,"startColumn":0,"endColumn":0}}],"reason":"autoLogPoints"},{"path":"/users/jeltehomminga/coding/my-luminus-web/middleware.ts","autoLogPoints":[{"range":{"startLine":350,"endLine":350,"startColumn":0,"endColumn":0}}],"reason":"autoLogPoints"},{"path":"/users/jeltehomminga/coding/my-luminus-web/src/screens/dashboard/move-track-and-trace-cta.tsx","autoLogPoints":[{"range":{"startLine":16,"endLine":16,"startColumn":0,"endColumn":0}}],"reason":"autoLogPoints"},{"path":"/users/jeltehomminga/coding/my-luminus-web/src/components/address-form-part/utils/parse-postal-code-and-city.ts","autoLogPoints":[{"range":{"startLine":52,"endLine":52,"startColumn":0,"endColumn":0}}],"reason":"autoLogPoints"},{"path":"/users/jeltehomminga/coding/my-luminus-web/app/api/current-contract-usps/[ean]/route.ts","autoLogPoints":[{"range":{"startLine":6,"endLine":6,"startColumn":0,"endColumn":0}}],"reason":"autoLogPoints"},{"path":"/users/jeltehomminga/coding/my-luminus-web/app/api/account/active-business-partner/route.ts","autoLogPoints":[{"range":{"startLine":30,"endLine":30,"startColumn":0,"endColumn":0}}],"reason":"autoLogPoints"},{"path":"/users/jeltehomminga/coding/my-luminus-web/src/screens/contracts-and-services/service-block.tsx","autoLogPoints":[{"range":{"startLine":16,"endLine":16,"startColumn":0,"endColumn":0}}],"reason":"autoLogPoints"},{"path":"/Users/jeltehomminga/coding/my-luminus-web/src/hooks/useMeterReadingsForEan/index.ts","reason":"open"},{"path":"/Users/jeltehomminga/coding/my-luminus-web/mocks/handlers.ts","reason":"open"},{"path":"/Users/jeltehomminga/coding/my-luminus-web/src/screens/contracts-and-services/contract-block.tsx","reason":"open"},{"path":"/Users/jeltehomminga/coding/my-luminus-web/src/components/chart/graph/index.tsx","reason":"open"},{"path":"/Users/jeltehomminga/coding/my-luminus-web/src/providers/MonitoringProvider/index.tsx","reason":"open"},{"path":"/Users/jeltehomminga/coding/my-luminus-web/src/screens/consumptions/details/index.tsx","reason":"open"},{"path":"/Users/jeltehomminga/coding/my-luminus-web/src/components/chart/index.tsx","reason":"open"}],"debug":false,"config":{"dateTimeFormat":"HH:mm:ss.SSS","dateTimeDisplayMode":"Absolute","installBinToPath":true,"openLinksInEditor":true,"maxLogViewerEntries":15000,"showWelcomeMessageInTools":true,"fontSize":10,"outputMode":"Beside File","captureFunctions":true,"searchUrl":"https://www.google.com/search?q={query}","enableProFeatures":true,"toolsToAutoPatch":["vite","jest","webpack","next.js","nest.js","cypress","http-server","serve","live-server","serverless"],"showErrors":true,"showLogsOnlyFromOpenedFiles":false,"showLogsFromLogPointsOnly":false,"showLogsFromMultipleRunningToolsTogether":false,"predictiveLoggingEnabled":true,"allowConsoleLogToTriggerPredictions":true,"showNetworkRequests":true},"allowedHosts":["localhost","127.0.0.1","example.cypress.io","Jeltes-MacBook-Pro.local","192.168.1.15"],"host":"127.0.0.1","forcedStart":false,"startTime":1732174430928}
08:33:53.785 info host starting tools discovery and integration
08:33:53.786 info host creating /Users/jeltehomminga/.console-ninja/.bin
08:33:53.824 info host installing /Users/jeltehomminga/.console-ninja/.bin/loader.js
08:33:53.833 info host installing /Users/jeltehomminga/.console-ninja/.bin/console-ninja
08:33:53.836 info host looking for tools to patch
08:33:54.656 info host found 12 tool file(s) to patch
08:33:54.657 info host patching /users/jeltehomminga/coding/my-luminus-web/node_modules/cypress/lib/cypress.js
08:33:54.658 info host patching /users/jeltehomminga/coding/my-luminus-web/node_modules/jest/bin/jest.js
08:33:54.658 info host patching /users/jeltehomminga/coding/my-luminus-web/node_modules/jest-runner/build/testWorker.js
08:33:54.658 info host patching /users/jeltehomminga/coding/my-luminus-web/node_modules/next/dist/compiled/webpack/bundle5.js
08:33:54.658 info host patching /users/jeltehomminga/coding/my-luminus-web/node_modules/next/dist/build/webpack/loaders/next-swc-loader.js
08:33:54.658 info host patching /users/jeltehomminga/coding/my-luminus-web/node_modules/webpack/lib/WebpackOptionsApply.js
08:33:54.658 info host patching /users/jeltehomminga/coding/my-luminus-web/node_modules/webpack/lib/index.js
08:33:54.658 info host patching /users/jeltehomminga/coding/my-luminus-web/node_modules/webpack/lib/webpack.js
08:33:54.724 info host Host's startup and patch time: 3796 ms
08:33:54.724 info host completed tools discovery and integration
08:33:54.732 ERR! host uncaught exception: No clients connected
08:34:06.859 info buildHook-36687 installing build hook for next.js at /users/jeltehomminga/coding/my-luminus-web, node v22.11.0
08:34:06.918 info host client connected: buildHook { address: '127.0.0.1', family: 'IPv4', port: 54007 }
08:34:06.989 WARN host ✘ node v22.11.0, and next.js v14.2.18 are not yet supported in the Community edition of Console Ninja.
08:34:06.989 WARN host We are working hard on it for you https://tinyurl.com/3h9mtwra.
08:34:06.989 WARN host 
08:34:06.989 WARN host Estimated release dates:
08:34:06.989 WARN host   - Community users: around 13th January, 2025 (subject to team availability)
08:34:06.989 WARN host   - PRO users:       priority access is available now
08:34:07.630 info buildHook-36697 installing build hook for next.js at /users/jeltehomminga/coding/my-luminus-web, node v22.11.0
08:34:07.680 info host client connected: buildHook { address: '127.0.0.1', family: 'IPv4', port: 54007 }
08:34:07.688 WARN host ✘ node v22.11.0, and next.js v14.2.18 are not yet supported in the Community edition of Console Ninja.
08:34:07.688 WARN host We are working hard on it for you https://tinyurl.com/3h9mtwra.
08:34:07.688 WARN host 
08:34:07.688 WARN host Estimated release dates:
08:34:07.688 WARN host   - Community users: around 13th January, 2025 (subject to team availability)
08:34:07.688 WARN host   - PRO users:       priority access is available now
08:36:53.442 info host client disconnected: buildHook 1006
08:36:53.443 info host client disconnected: buildHook 1006, has clients true
08:36:53.464 info host client disconnected: buildHook 4100
08:36:53.464 info host client disconnected: buildHook 4100, has clients false
08:36:53.926 info buildHook-38036 installing build hook for next.js at /users/jeltehomminga/coding/my-luminus-web, node v22.11.0
08:36:53.982 info host client connected: buildHook { address: '127.0.0.1', family: 'IPv4', port: 54007 }
08:36:53.998 WARN host ✘ node v22.11.0, and next.js v14.2.18 are not yet supported in the Community edition of Console Ninja.
08:36:53.998 WARN host We are working hard on it for you https://tinyurl.com/3h9mtwra.
08:36:53.998 WARN host 
08:36:53.998 WARN host Estimated release dates:
08:36:53.998 WARN host   - Community users: around 13th January, 2025 (subject to team availability)
08:36:53.998 WARN host   - PRO users:       priority access is available now
08:36:54.501 info host client disconnected: buildHook 4100
08:36:54.501 info host client disconnected: buildHook 4100, has clients false

```

**Comment by smcenlly** [comment]
> I cannot find an activation screen where I can enter my activation key, do you have a link to that screen?

1. In VS Code, run the `Console Ninja: Open Start View` command.
2. Click on the `Licensing` section on the left hand side of the screen:

<img width="238" alt="image" src="https://github.com/user-attachments/assets/38d3e380-a2e8-48d0-9ac9-37dde20b0178">

3. You should see a screen similar to this (note: you may need to select to activate your license:
<img width="888" alt="image" src="https://github.com/user-attachments/assets/b72c6648-85e5-482d-a954-860e12f903df">
4. Instead of providing your email address, provide your license key.

**Comment by jeltehomminga** [comment]
I get a message that I need to purchase PRO, 

![CleanShot 2024-11-22 at 05 22 41](https://github.com/user-attachments/assets/b7a9eafb-044a-4617-b1d7-b41413a37dc0)

when I try that I get message the email is already registered. See earlier message above

So looks like there is no way in the ui to resume or activate the license at the moment

**Comment by smcenlly** [comment]
Sorry for the confusion. There was a bug in our user interface that led me to believe that your license was valid.

Our Wallaby Account was not allowing you to restart your subscription if your license was expired. This has been fixed.

Please log into your Wallaby Account. You should be able to restart your subscription using the `Restart Subscription` option:

<img width="418" alt="image" src="https://github.com/user-attachments/assets/8d80af98-ff99-4ba0-bb17-a1eadbf90adb">

Sorry for the inconvenience while we worked this out.

--

If you're already logged into your Wallaby Account, you should be able to Ctrl/Cmd + Refresh the page to see the `Restart Subscription` option.

**Comment by jeltehomminga** [comment]
ok, thanks, yeah I now saw this option. after clicking it I first got an error, but after that I was logged out and when I log in again I can see my subscription was restarted for a year, 

![CleanShot 2024-11-22 at 06 30 04](https://github.com/user-attachments/assets/8db64535-8f4b-4e3c-a8fd-5f0cdad4868a)

Would be nice to be able to choose also for a monthly subscription if you want to reactivate your subscription.  
But everything is working now. Thanks for the support
